src/alchemlyb/parsing/gmx.py

```diff
@@ -29,6 +26,11 @@ def extract_u_nk(xvg, T):
     u_nk : DataFrame
         Potential energy for each alchemical state (k) for each frame (n).
 
+
```

Comment:
```suggestion

    Note
    -----
    Previous versions of alchemlyb (<0.5.0) used the `GROMACS value of the molar gas constant
    <https://manual.gromacs.org/documentation/2019/reference-manual/definitions.html>`_
    of  :math:`R = 8.3144621 \times 10^{−3}\, \text{kJ}\cdot\text{mol}^{-1}\cdot\text{K}^{-1}`
    instead of the scipy value :data:`scipy.constants.R` (see :mod:`alchemlyb.postprocessors.units`).
    The relative difference between the two values is :math:`6 \times 10^{-8}`. 
    
    Therefore, results in :math:`kT` for GROMACS data will differ between alchemlyb ≥0.5.0 and
    previous versions; the difference is on the order of :math:`10^{-7}` for typical cases.
    
    
```